packages/e2e/cypress/integration/common/storagebrowser.ts

```diff
@@ -34,3 +34,34 @@ When(
     });
   }
 );
+
+Then('I see download for file {string}', (name: string) => {
+  cy.contains('table tbody td:nth-child(2)', new RegExp('^' + name + '$'))
+    .siblings()
+    .last()
+    .children('button');
```

Comment:
Is this the simplest way to locate this download button?

Reply:
Unfortunately yes. This kinda also checks you are clicking download of that particular row